content/en/blog/_posts/2023-03-08-nodeport-dynamic-and-static-allocation.md

```diff
@@ -0,0 +1,173 @@
+---
+layout: blog
+title: "Kubernetes 1.27: Avoid Collisions Assigning Ports to NodePort Services"
+date: 2023-04-10
```

Comment:
Please change this date and the filename according to the suggestion.

Reply:
thanks, it's done